implementations/rust/ockam/ockam_node/src/runtime.rs

```diff
@@ -14,3 +14,106 @@ pub fn take() -> Runtime {
         .take()
         .expect("Runtime was consumed")
 }
+
+#[cfg(feature = "watchdog")]
+pub(crate) mod watchdog {
+    use std::collections::HashMap;
+    use std::thread::ThreadId;
+    use std::time::Duration;
+    use tokio::runtime::Runtime;
+
+    struct ThreadInfo {
+        poll_count: u64,
+        was_parked: bool,
```

Comment:
do you think `is_parked: bool,` makes more sense?

Reply:
`was_parked` can be understood as `was_ever_parked`, or maybe it's just me

Reply:
I went with `parked_on_previous_iteration`